ifrs17/Test/ImportStorageTest.ipynb

```diff
@@ -390,7 +390,46 @@
                 "\n  };",
                 "\n",
-                "\nvar activity = await CheckIfrsVariablesFromImportStorageAsync(ImportFormats.Actual, inputDataSetForWorkspace, inputDataSetForDataSource, ivsBenchmark);",
+                "\nvar activity = await CheckIfrsVariablesFromImportStorageAsync(ImportFormats.Actual, inputDataSetForWorkspace, inputDataSetForDataSource, ivsBenchmark, args);",
+                "\nactivity"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "activity.Status.Should().Be(ActivityLogStatus.Succeeded);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "//There is a previous year, and for the current year Actuals are imported for a given Scenario",
+                "\nvar basicIfrsVariable = new IfrsVariable{Partition = partition.Id, DataNode = groupOfInsuranceContracts, AccidentYear = null, AmountType = AmountTypes.PR, Novelty = Novelties.C, EstimateType = EstimateTypes.AA};",
+                "\nvar inputDataSetForDataSource = new IfrsVariable[]{",
+                "\n    basicIfrsVariable with {AocType = AocTypes.BOP, EstimateType = EstimateTypes.DA, Value = 1000.0},",
+                "\n    basicIfrsVariable with {AocType = AocTypes.BOP, Value = 100.0},",
+                "\n    basicIfrsVariable with {AocType = AocTypes.CL, Value = 150.0},",
+                "\n  };",
+                "\n",
+                "\nvar inputDataSetForWorkspace = new IfrsVariable[]{",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AocType = AocTypes.CL, Value = -15.0},",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AmountType = AmountTypes.CL, AocType = AocTypes.CL, Value = -99.0},",
+                "\n  };",
+                "\n",
+                "\nvar ivsBenchmark = new IfrsVariable[]{",
+                "\n    basicIfrsVariable with {Partition = partition.Id, AocType = AocTypes.BOP, EstimateType = EstimateTypes.DA, Value = 1000.0},",
+                "\n    basicIfrsVariable with {Partition = partition.Id, AocType = AocTypes.BOP, Value = 100.0},",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AocType = AocTypes.CL, Value = -15.0},",
+                "\n    basicIfrsVariable with {Partition = partitionScenarioMTUP.Id, AmountType = AmountTypes.CL, AocType = AocTypes.CL, Value = -99.0},",
+                "\n  };",
+                "\n",
+                "\nvar activity = await CheckIfrsVariablesFromImportStorageAsync(ImportFormats.Actual, inputDataSetForWorkspace, inputDataSetForDataSource, ivsBenchmark, argsScenarioMTUP);",
```

Comment:
Interesting test cases are: 
Cashflow format for ScenarioX with primaryScope GricX with underlying GIC already imported BE (only and ScenarioX) too.
The Storage should contain IFRS of GIC (secondary scope) (of BE and ScenarioX, respectively).